diagnostic.md

```diff
@@ -5,27 +5,29 @@ Record your responses inside the fenced code blocks below each question.
 1.  Give an example of a visual hierarchy that could be modeled with components. Explain why each piece might be it's own component.
 
     ```md
-    <!-- your response here -->
+    Build a Pizza! You have 3 crusts representing a template. Each series of toppings per crust is a component.
```

Comment:
Any UI elements that are nested are a good answer. Siblings do not count.

For example: There is a visual hierarchy in the navbar in many applications.
The navbar represents a block and buttons or links within that block are
elements of the navigation structure. Another example might be a sidebar for
a blog, which is a visual block containing navigation elements or even
comments.

0/1